Examples/Image/Detection/FastRCNN/A1_GenerateInputROIs.py

```diff
@@ -7,14 +7,15 @@
 from __future__ import print_function
 from builtins import input
 import os, sys, datetime
+import cv2
```

Comment:
It is being used [here](https://github.com/rotimpe/CNTK/blob/2f40b8a2e0d4c589051e844426f136a0d5d27635/Examples/Image/Detection/FastRCNN/A1_GenerateInputROIs.py#L64)